src/client/inkeep/chat-button/inkeep-chat-button.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
```

Comment:
Please move the Inkeep code inside `src/client/analytics`

Reply:
@JuroUhlar inkeep is not really "analytics". It is mostly a trird-party integration, I didn't move there because it could be confusing but I indeed think it should be inside client.